apps/Minecraft Pi (Modded)/install

```diff
@@ -3,12 +3,18 @@
 
 # Remove old MCPILs
 sudo apt-get remove -y mcpil-r mcpil &>/dev/null || true
+
+# Install the repos
 wget -qO- https://raw.githubusercontent.com/MCPI-Revival/mcpi-packages/master/install.sh | grep -v 'sudo apt update' | sudo bash - || error "Failed to install APT repo!"
+curl -s --compressed "https://mcpiscript.github.io/PPA/KEY.gpg" | sudo apt-key add -
+sudo curl -s --compressed -o /etc/apt/sources.list.d/mcpiscript.list "https://mcpiscript.github.io/PPA/mcpiscript.list"
+sudo apt update
+
```

Comment:
Are these lines only used if the Planet Launcher option is chosen? If so, please put them in an `if` statement.